scripts/actions/mobskills/decussate.lua

```diff
@@ -20,7 +20,7 @@ end
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local dmgmod = 1.2
     local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg() * 3, xi.element.EARTH, dmgmod, xi.mobskills.magicalTpBonus.NO_EFFECT)
-    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.MAGICAL, xi.damageType.EARTH, math.random(2, 3) * info.hitslanded)
+    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.MAGICAL, xi.damageType.EARTH, math.random(2, 3))
```

Comment:
The `info.hihtslanded` should be removed anyway but just as a note (does not have to be in this PR) math.random for magical moves seems extremely odd. As far as I am aware this is a huge AOE skill for both NMs that use this. It probably wipes all shadows like all the other magical moves (assuming its magical). We found a lot of skills that were listed wrong the wiki